_security-analytics/usage/correlation-graph.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Working with the correlation graph
+parent: Using Security Analytics
+nav_order: 45
+---
+
+# Working with the correlation graph
+
+The correlation engine is an experimental feature released in OpenSearch 2.7. Therefore, we do not recommend using the feature in a production environment at this time. For updates on the progress of correlation engine, see [Security Analytics Correlation Engine](https://github.com/opensearch-project/security-analytics/issues/369) at GitHub. To share ideas and provide feedback, join the [Security Analytics forum](https://forum.opensearch.org/c/plugins/security-analytics/73).
+{: .warning }
+
+The correlation graph is a security findings knowledge graph. It provides a visualization for information generated by the correlation engine and allows you to focus on specific correlations and inspect them in greater detail. Information in the graph includes findings by log type, the severity levels for the findings, the correlations drawn between findings, and the relevance of the correlations, among other details. You can also manipulate the graph to gain further insights into specific events of interest. This includes filtering findings by date and time, zooming in on the relationship between specific findings and their correlations, and filtering by log type and severity level. Use this section to learn more about using the graph. 
+
+## Acccessing the graph
+
+Begin by selecting **Security Analytics** in the Dashboards home page menu. Then select **Correlations** from the Security Analytics menu on the left side of the screen. The **Correlations** page is displayed.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph.png" alt="The correlation graph" width="85%">
+
+## Interpreting the graph
+
+The graph displays findings as nodes with colored borders expressing their severity level. A three-letter abbreviation inside the node indicates the log type. The lines that connect the findings represent the correlations between them. A heavy line indicates a strong correlation, while a light line shows a weaker connection.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph-detail.png" alt="The correlation graph" width="40%">
+
+## Using the graph
+
+You can control which findings are displayed in the graph by filtering by severity level, log type, and time filter. The time filter controls the findings that appear in the graph by limiting the dates on which the findings were generated.
```

Comment:
"on" the graph (all instances in this section)? "by limiting the dates on which the findings were generated" is slightly unclear. Can we simplify?